doc/Release_Build_Creation.rst

```diff
@@ -30,7 +30,7 @@ Procedure
 
    ::
    
-    time docker run --rm -v /var/artifacts:/var/artifacts -v /root/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:centos-7.8.2003 make clean build
+    docker run --rm -v /var/artifacts:/var/artifacts -v /root/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:centos-7.8.2003 make clean build
```

Comment:
Yes I understand, it can make us like we don't really know what we are doing, I guess. @johnbent what was your reasoning behind removing this again? I think a note saying that time is unnecessary and that the command will take upwards of an hour to complete is the solution.

Reply:
It seems weird that the instruction says:

*   Use docker to build your software
*   And then shows 'time docker . . . '

It makes it seem like the author was just given some instructions and doesn't understand them.  So, it's cool to leave 'time' but then we should change the instructions to say, "Use docker to build the software (or whatever it says).  Optionally prefix it with time which can be useful to measure how long the build takes."